<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form</class>
 <widget class="QWidget" name="form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0,0">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>220</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Настройка ядра</string>
          </property>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>91</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Масса:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>116</y>
             <width>161</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Координата x,y,z:</string>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="M">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>43</y>
             <width>121</width>
             <height>22</height>
            </rect>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="X">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>137</y>
             <width>61</width>
             <height>22</height>
            </rect>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>67</y>
             <width>91</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Площадь:</string>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="S">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>121</width>
             <height>22</height>
            </rect>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
          </widget>
          <widget class="QLabel" name="label_11">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>47</y>
             <width>47</width>
             <height>13</height>
            </rect>
           </property>
           <property name="text">
            <string>кг</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_12">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>95</y>
             <width>47</width>
             <height>13</height>
            </rect>
           </property>
           <property name="text">
            <string>м²</string>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="Y">
           <property name="geometry">
            <rect>
             <x>71</x>
             <y>137</y>
             <width>61</width>
             <height>22</height>
            </rect>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="Z">
           <property name="geometry">
            <rect>
             <x>132</x>
             <y>137</y>
             <width>61</width>
             <height>22</height>
            </rect>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="xV">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>181</y>
             <width>61</width>
             <height>22</height>
            </rect>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>160</y>
             <width>161</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Скорость x,y,z (м/с):</string>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="zV">
           <property name="geometry">
            <rect>
             <x>131</x>
             <y>181</y>
             <width>61</width>
             <height>22</height>
            </rect>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="yV">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>181</y>
             <width>61</width>
             <height>22</height>
            </rect>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Внешние условия</string>
          </property>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>101</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Плотность среды:</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="p">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>121</width>
             <height>22</height>
            </rect>
           </property>
           <property name="maximum">
            <number>10000000</number>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="xVv">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>91</width>
             <height>22</height>
            </rect>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>141</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Скорость ветра x,y  (м/с):</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_17">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>44</y>
             <width>47</width>
             <height>13</height>
            </rect>
           </property>
           <property name="text">
            <string>кг/м³</string>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="yVv">
           <property name="geometry">
            <rect>
             <x>103</x>
             <y>90</y>
             <width>91</width>
             <height>22</height>
            </rect>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="start">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Старт</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stop">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Пауза</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QChartView" name="graphicsView_1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>525</width>
          <height>525</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QChartView" name="graphicsView_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>525</width>
          <height>525</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="status">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="x_axis">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>250</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string> Y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="y_axis">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>250</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>365</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>XY</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="xy_axis">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>250</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="z_axis">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>250</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
